ide-support/revliburl.livecodescript

```diff
@@ -2058,7 +2058,7 @@ function ulBuildHttpRequest pUrl
          end if
       end repeat
       if laSocketSecured[tSocket] is false then
-         put "CONNECT" &&  laUrlFormat[pUrl]["urlHost"] & ":" & laUrlFormat[pUrl]["urlPort"] && "HTTP/1.1" & cr & "Host:" && laUrlFormat[pUrl]["urlHost"] & ":" & laUrlFormat[pUrl]["urlPort"] into tRequest
+         put "CONNECT" &&  laUrlFormat[pUrl]["urlHost"] & laUrlFormat[pUrl]["urlPort"] && "HTTP/1.1" & cr & "Host:" && laUrlFormat[pUrl]["urlHost"] & laUrlFormat[pUrl]["urlPort"] into tRequest
```

Comment:
@montegoulding @trevordevore I think this is okay.  IPv6 HTTP 1.1 requests look something like:

```
CONNECT [::1]:443 HTTP/1.1
Host: [::1]:443
User-Agent: curl/7.43.0
Proxy-Connection: Keep-Alive
```

As long as we ensure that `laUrlFormat[pUrl]["urlHost"]` continues to end with a ":" when IPv6 support is added, I think that this code can be left how it is.